pkg/document/change/id.go

```diff
@@ -83,24 +97,44 @@ func (id ID) NewTimeTicket(delimiter uint32) *time.Ticket {
 	)
 }
 
-// SyncLamport syncs lamport timestamp with the given ID.
-// https://en.wikipedia.org/wiki/Lamport_timestamps#Algorithm
-func (id ID) SyncLamport(otherLamport int64) ID {
+// SyncClocks syncs logical clocks with the given ID.
+func (id ID) SyncClocks(other ID) ID {
+	lamport := id.lamport + 1
+	if id.lamport < other.lamport {
+		lamport = other.lamport
```

Comment:
I have a question about this function. Is the version vector being used here Lamport-synced?

If so, shouldn't the update logic be `other.lamport + 1` instead?

Reply:
Also, I think `new ID` should consider clients other than these two.
If I'm wrong, or if it seems like we're thinking of different ideas, please let me know.